### ISSUE TITLE
Needed to run `valet link` after creating site

### ISSUE THREAD

**Issue by jjeaton** [aaemnnosttv/wp-cli-valet-command#22]
I'm not using `valet park` for all sites in a directory, so possibly that's the issue here.

After running `wp valet new <sitename>` I needed to cd into the `<sitename>` directory and run `valet link <sitename>` before it works.

Is this expected?

**Comment by aaemnnosttv** [comment]
Yes, the command currently assumes you're using it in a "parked" directory.

I've thought about adding the option to, or just linking it by default though. I always use it in a parked directory so I'm leaning towards the option.

**Comment by jjeaton** [comment]
Yeah a `--link` option might be nice.

**Comment by javorszky** [comment]
Hm, if you're using it in parked mode, then that could create inconsistent sites.

```
$ mkdir wpvalettest
$ cd wpvalettest
$ wp valet new wpvaletfoo
```
Valet parked will assume the site is in `https://wpvalettest.tld`, yet we just set it to be `https://wpvaletfoo.tld`.

**Comment by aaemnnosttv** [comment]
If we were to implement the `--link` flag, it would be optional. By default, the command has always assumed you're using a parked directory or linking it yourself, this would just be a little quicker for those who are doing an install in a non-parked directory. 

Of course the command would also link the proper directory, that's not a problem 😄

**Comment by idleberg** [comment]
Any news on a `--link` flag?
